Integrations/KennaV2/KennaV2_description.md

```diff
@@ -0,0 +1,2 @@
+Kenna is a Risk Intelligence & Vulnerability platform that enables InfoSec
+  teams to prioritize and remediate vulnerabilities faster.
```

Comment:
please write here the instance config:
https://github.com/demisto/content/blob/master/Integrations/Palo_Alto_Networks_WildFire_v2/Palo_Alto_Networks_WildFire_v2_description.md
also supports MD.